utils/rtty/files/rtty.init

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh /etc/rc.common
+
+. /lib/functions/network.sh
+
+USE_PROCD=1
+START=99
+
+BIN=/usr/sbin/rtty
+
+start_rtty() {
+    local cfg="$1"
+    local interface ifname id description host port ssl
+
+    uci_validate_section rtty rtty "${1}" \
+        'interface:uci("network", "@interface"):lan' \
```

Comment:
This hard codes the interface name. It seems that this is not good practice.

Reply:
@poranje , I agree.  Personally I also prefer such config options being explicit in the surface without needing to dive into the code.

@zhaojh329 In case it may help, it's now also a convention to add package specific doc in `rtty/README.md`.  Contents such as "when both `interface` and `id` are specific which will take precedence" can go there.

Reply:
There is a description in the default configuration file

# You must specify an interface or id,
# If the id is not specified, RTTY will uses the MAC address
# of the interface you specify as its ID, otherwise the id
# you specify is used as its ID